src/index.html

```diff
@@ -7,8 +7,72 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="image">
+        <img src="/src/images/logo.png"
+        alt="logo"
+        class="logo__image">
```

Comment:
```suggestion
        <img
          src="/src/images/logo.png"
          alt="logo"
          class="logo__image"
        >
```

Reply:
every attribute should be on new line if you started moving it(with +2 indentation)

You can move them if you have more than 2 attrs

Reply:
fix everywhere please